scripts/globals/mobskills/howling_fist.lua .lua

```diff
@@ -0,0 +1,26 @@
+---------------------------------------------
```

Comment:
This file is named: `howling_fist.lua .lua ` 👀

Reply:
luaception lol. fixed =)